### PR TITLE
feat: portal loop's main contracts and gnoweb improvements

### PR DESCRIPTION
**Pull request by moul**
Addresses #1131

## Completed Tasks

-  [x] Completed the home page by adding static content and blocks.
-  [x] Implemented support for redirects and aliases to improve SEO.
-  [x] Made some improvements to p/demo/ui, but more work is still needed.
-  [x] Enhanced p/demo/blog by adding widget support.
-  [x] Transferred previous static webpages to realms.
-  [x] Created a new personal realm `p/manfred/present` for a Gno-powered presentation.
-  [x] Refactored gnoweb to remove static pages, improve maintainability and design consistency, and added optional analytics.

## Next Steps after Merging

- Anyone:
  - Improve r/gnoland/home.
  - Create and enhance additional `p/demo/*` pages to simplify maintenance of `r/gnoland/home`.
  - Start writing personal and team realms to incorporate more dynamic data on-chain. Consider adding these realms as widgets on the homepage.
  - Encourage individuals to create dedicated realms, preferably dynamic ones. Then, import these new realms into the homepage to include widgets such as "upcoming events."
- Manfred:
  - Develop dynamic contracts for Worxdao, including project, team, and people directories. Also, implement DAO features v0 and contributor profiles.

### CODE REVIEW

**Review by gfanton — APPROVED** [review]
There are some links that are broken, but I don't think they are directly linked to this PR. Otherwise, LGTM

**Review by zivkovicmilos — APPROVED** [review]
Looks great 💯 

Love the widget idea

File: examples/gno.land/r/gnoland/home/home.gno [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+	return ui.Element{
+		ui.H3("Explore New Packages and Realms"),
+		ui.Columns{
+			3,
+			[]ui.Element{
+				{
+					ui.H4("r/gnoland"),
+					ui.BulletList{
+						ui.Link{URL: "r/gnoland/blog"},
+						ui.Link{URL: "r/gnoland/dao"},
```
[the remark this answers is not in the document]
Comment (moul) [reply examples/gno.land/r/gnoland/home/home.gno:166]: Let's utilize the portal loop effect to have someone (perhaps you) open different PRs solely using gnoweb. They can fix issues while browsing.

File: gno.land/cmd/gnoweb/main.go [hunk 2]
```diff
@@ -217,6 +260,7 @@ func handlerRealmMain(app gotuna.App) http.Handler {
 				writeError(w, err)
 				return
 			}
+			println(string(res.Data))
```
[the remark this answers is not in the document]
Comment (moul) [reply gno.land/cmd/gnoweb/main.go:263]:
```suggestion
```

**Pull request merged**